ZackHolmberg/COMP4350-Project
# Implemented signing in frontend and backend, updated dockerfile for transactions

how to test:
build and run containers as per usual to test.

sample payload to test with postman:

`{
    "from": "-----BEGIN PUBLIC KEY-----\nMIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDnO7/dQaeEsLKa4KN703PR1JDx\nBbFAi10+ofEo0PeFUKgRdDt5tBUx/XrfCpiZmLTNQcRDBtNXadtp5TaBgeyb0N2w\n2TwCElB51bpJhn/haTwkXLipxnYdZkYtdvJ03SxYZ9HBUXhnCcTVfgFDHabLhocd\nYJ6CNECveFJr0mYm4wIDAQAB\n-----END PUBLIC KEY-----",
    "to" : "687",
    "amount": 10,
    "id" : "d94877994bf39b0bd2cbec2547d13e8650a5141b3e070ef7d7169ef89bc9732e",
    "signature" : "d72eb3a1afcd9f06b2127bbe7f8166869686630db2bf9b9ac1f41074d06fbaa5765addfa75ddaffd4c1854687c26ed960a1bff590f95ddc3a385a92b362f8d4ea4226b4531c8482a2f2432c062340b4f70d57f42db0733637d844d05c542572c8d9bed954a9afa244e704b4ac5d087b664a60b7b7bf316760c04b7a720f04803"   
}`

you can try altering the id or signature and it will give you an error message stating signature verification failure.

on successful runs, you should get a wallet verification failure
